doc/ruleSets/Symfony.rst

```diff
@@ -37,6 +37,9 @@ Rules
 - `general_phpdoc_tag_rename <./../rules/phpdoc/general_phpdoc_tag_rename.rst>`_
   config:
   ``['replacements' => ['inheritDocs' => 'inheritDoc']]``
+- `global_namespace_import <./../rules/phpdoc/general_phpdoc_tag_rename.rst>`_
```

Comment:
would you mind to run https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/dev-tools/doc.php on your PR?

Reply:
nice thanks! done